MDConfigDebug.ini

```diff
@@ -15,6 +15,7 @@ GameSynchronizerEnabled=true
 UseSceneBuffer=true
 GameClockActive=true
 LogDirectory="user://logs/"
+LogConsoleToGDPrint=true
```

Comment:
You should also add this setting to MDConfigExport.ini, I also noticed that I forgot to copy the latest settings from MDConfigDebug.ini into the export file. So maybe you could just copy all of them in the scope of this PR and set some sensible defaults for exported games.

Reply:
done